fbw-a32nx/src/systems/instruments/src/SD/Pages/Eng/Eng.tsx

```diff
@@ -130,35 +138,42 @@ const PressureGauge = ({ x, y, engineNumber }: ComponentPositionProps) => {
             <line className="GaugeMarking" x1={x} y1={y} x2={x} y2={y + 5} />
             <line className="GaugeMarking" x1={x + 45} y1={y + 50} x2={x + 51} y2={y + 50} />
             <Arc x={x} y={y + 50} radius={50} toValue={100} scaleMax={100} className="WhiteLine NoFill" />
-            <Arc x={x} y={y + 50} radius={50} toValue={OIL_PSI_VLOW_LIMIT} scaleMax={100} className="RedLine NoFill" />
-            <Needle
-                x={x}
-                y={y + 50}
-                length={60}
-                scaleMax={100}
-                value={getNeedleValue(engineOilPressure, OIL_PSI_MAX)}
-                className={`NoFill ${needleClassName}`}
-                dashOffset={-40}
-            />
-            <text
-                x={x}
-                y={y + 45}
-                className={`FontLarge TextCenter ${textClassName}`}
-            >
-                {displayedEngineOilPressure}
-            </text>
+            <g visibility={activeVisibility}>
```

Comment:
In react, you could use conditional rendering to hide/show that part.

e.g.

```ts
{activeVisibility && (<.....>)}
```

But I think you can leave it like this, should even be a bit better as we do not manipulate the DOM then.

Reply:
I used the same concept as on the EWD

Reply:
Yep, but EWD uses the MSFS avionics framework which works a bit different. Anyway, just wanted to make you aware :) It's fine as is.

Reply:
> In react, you could use conditional rendering to hide/show that part.
> 
> e.g.
> 
> ```ts
> {activeVisibility && (<.....>)}
> ```
> 
> But I think you can leave it like this, should even be a bit better as we do not manipulate the DOM then.

You can do that in the avionics framework but it’s quite expensive to add/remove things from the DOM so it’s best to avoid that.